en/scripting/typescript.md

```diff
@@ -162,7 +172,24 @@ export class MyUser extends cc.Component {
 
 When you enter `this.myModule.`, you will be prompt the properties declared in `MyModule.ts`.
 
-![Auto complete](assets/auto-complete.gif)
+![Auto complete](assets/auto-complete.gif) <br>
+__If you find that changing a declared property to an array type doesn't take effect in the editor, Please remove the binding component from the current node and rebind.__
```

Comment:
If the declared property is modified to an Array type, but it does not take effect in the editor. Please remove the component that is bound by the current node, and then add it again.